### DISCUSSION TITLE
Notes lost after hitting "Ctrl+s"

### DISCUSSION THREAD

**Discussion by AWP-01** [sumatrapdfreader/sumatrapdf#3514] [Q&A] [answered]
Hello everyone,
    I really appreciate your efforts to create such an opensource pdf reader! I really enjoy using it to read research papers, however, I encounter some problem and need help.
    After taking some notes in my pdf file, for example, some highlights, I would hit "Ctrl + s" unconsciously. The notes I took just a moment ago will disappear when I confirm the replacement of the original file. Hitting the exit icon and saving the file has no problem. I usually hit "Ctrl + s" to save my notes when reading long papers and this problem occurs occasionally.  
    Hope someone can help me, thank you! 
    I'm in Windows 11 Home (Chinese) and SumatraPDF 3.4.6 64bit.

**Comment by GitHubRulesOK** [comment]
unsure how it would be best used but you can set the key actions so if you want to deliberately BLOCK Ctrl+S set it to null or if you want it to save set CTRL+S to the shifted command

see https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts

**Reply by AWP-01** [reply]
Thank you for your reply! However, my problem is that when I hit "Ctrl + s" to save my notes, for example some highlights, and save the file according to the prompt window, the highlight notes disappear after saving. I have the habit of saving my notes while reading a paper and this behavior loses my notes. It happens occasionally. And when I save the file by clicking the close icon, everything goes well. It's not a very big problem for me, but I am just curious about this. Thank you again.

1. Make some notes in the file:  
<img width="718" alt="1" src="https://github.com/sumatrapdfreader/sumatrapdf/assets/34649355/a8d77701-7c74-4288-8bc8-70023e603b11">  

2. Hit "Ctrl + s" to save the notes:  
<img width="506" alt="3" src="https://github.com/sumatrapdfreader/sumatrapdf/assets/34649355/96943a1e-29a8-4acf-acc7-83837b087207">  

3. And the notes disappear after the previous operation:  
<img width="536" alt="4" src="https://github.com/sumatrapdfreader/sumatrapdf/assets/34649355/78972539-dbb9-460b-a88f-d204ddc31a69">

**Comment by GitHubRulesOK** [comment]
I suggest only latest pre-release behaviours can be altered  and any such issue may already have changed thus you need to see if that problem is now different by download current pre-release from https://www.sumatrapdfreader.org/prerelease

**Reply by AWP-01** [reply]
Thank you for your reply! I gave it a try and the problem still exists. Maybe it is related to my file or something else.

**Reply by GitHubRulesOK** [reply, marked as answer]
files can easily be corrupted when appending annotation as it alters the core structure and any small mistake has a lasting detrimental effect (always keep a copy of a source file and save as little as possible to maintain integrity/reduce numbers of risk of collateral damage. If you have a rogue file it may be possible to "fix" by removing annotations in a copy (and even possibly restore some by transfer) but that's not a simple task. 

My suggestion in the past for draughtsmen that lost much alterations when pressing the esc key was glue an upturned push pin to the escape key, its amazing how fast they forgot their keyboard habits.

Seriously simply set that key combination to CmdNone
```
Shortcuts [
	[
		Cmd = CmdNone
		Key = Ctrl + S
	]
]
```

**Reply by AWP-01** [reply]
OK, now I understand it. Thank you! I'll set it according to your suggestion.
